### ISSUE TITLE
Performance comparison of understand_sentiment model in Fluid and Paddle V2 (CPU version)

### ISSUE THREAD

**Issue by ranqiu92** [PaddlePaddle/Paddle#7054]
Training Set Pass accuracy

|Pass|Paddle V2|Fluid|
|----|---------|-----|
|0 | 0.82996 | 0.83179 |
|1	| 0.90116	| 0.9046 |
|2	| 0.93624 | 0.93804 |
|3	| 0.965 | 0.96223 |
|4	| 0.98148 | 0.98115 |
|5	| 0.98896	| 0.99128 |
|6	| 0.99348 | 0.99287 |
|7	| 0.99684 | 0.99335 |
|8	| 0.99624 | 0.99632 |
|9	| 0.99736 | 0.99724 |

Test set accuracy

|Pass|Paddle V2|Fluid|
|----|---------|-----|
|0	| 0.87668 | 0.88143
|1 | 0.88524 | 0.88827
|2	| 0.88555 |	0.88164
|3	| 0.87504 |	0.87699
|4	| 0.87095 |	0.8776
|5	| 0.87455 | 0.87691
|6	| 0.87752 |	0.87463
|7	| 0.88096 |	0.87643
|8	| 0.87703 |	0.87892
|9	| 0.87736	| 0.87691

#6650
